src/engine/ibus/ibus_engine.cpp

```diff
@@ -67,9 +67,9 @@ void ibus_table_sel_dec() {
   }
 }
 
-void ibus_update_suggest(Suggestion suggest) {
-  // Assign suggestions
-  suggestions = suggest;
+void ibus_update_suggest(Suggestion suggestions) {
+  // Assign suggestions // WHY?
+  // suggestions = suggest;
```

Comment:
https://github.com/OpenBangla/OpenBangla-Keyboard/blob/90c9e4f1d1e76e43a23dd4df7353916c7cdd547e/src/engine/ibus/ibus_engine.cpp#L32
Here we update the global Suggestion variable, which is used to commit candidate, updating pre-edit text etc. I know this is really a bad style, but I haven't yet spent much time to refactor the code. So, for now, we need to update the global variable.

Reply:
@Adyel Can you revert this change? This PR is mostly blocked because of this change.

Reply:
This extra variable does not do anything. @mominul.